tests/auxtel/test_latiss_cwfs_align.py

```diff
@@ -0,0 +1,512 @@
+                    msg="zern comparison", z=z, measured=meas_zerns[i], i=i
+                ):
+                    self.assertTrue(abs(z - meas_zerns[i]) <= zern_tol[i])
+            for i, rz in enumerate(results["rot_zerns"]):
+                with self.subTest(
+                    msg="rot-zern comparison", rz=rz, measured=rot_zerns[i], i=i
+                ):
+                    self.assertTrue(abs(rz - rot_zerns[i]) <= zern_tol[i])
+            for i, h in enumerate(results["hex_offset"]):
+                with self.subTest(
+                    msg="hexapod comparison", h=h, measured=hex_offsets[i], i=i
+                ):
+                    self.assertTrue(abs(h - hex_offsets[i]) <= hex_tol[i])
+
+            # Note that there is a known issue with the sign of the telescope
+            # offsets, omitting for now.
+            # for i, t in enumerate(results['tel_offset']):
+            #     if t != 0:
+            #         with self.subTest(t=t, i=i):
+            #             self.assertTrue(abs(t - tel_offsets[i])
+            #             / abs(tel_offsets[i]) <= 0.05)
+
+            # Check that hexapod offsets were applied -  too hard to keep
+            # track of how many calls should happen since
+            # it also calls this to take the intra/extra focal images
+            self.assertTrue(self.ataos.cmd_offset.callback.called)
+            # check that telescope offsets were applied twice
+            self.assertEqual(self.script.atcs.offset_xy.call_count, 2)
+            # check that total offsets are correct within the tolerance or 5%
+            #
+            self.assertTrue(
+                (
+                    abs(self.script.total_coma_x_offset - total_xcoma)
+                    / abs(total_xcoma)
+                    <= 0.05
+                )
+                or (abs(self.script.total_coma_x_offset - total_xcoma) < hex_tol[0])
+            )
+            self.assertTrue(
+                (
+                    abs(self.script.total_coma_y_offset - total_ycoma)
+                    / abs(total_ycoma)
+                    <= 0.05
+                )
+                or (abs(self.script.total_coma_x_offset - total_xcoma) < hex_tol[1])
+            )
+            logger.debug(
+                f"Measured total focus offset is {self.script.total_focus_offset:0.5f}"
+            )
+            logger.debug(f"Reference total focus offset value is {total_focus:0.5f}")
+            logger.debug(f"Tolerance is {max((0.05*total_focus, hex_tol[2])):0.5f}")
+            self.assertTrue(
+                (
+                    abs(self.script.total_focus_offset - total_focus) / abs(total_focus)
+                    <= 0.05
+                )
+                or (abs(self.script.total_focus_offset - total_focus) < hex_tol[2])
+            )
+
+    @unittest.skipIf(
+        DATA_AVAILABLE is False,
+        "Data not available for test_analysis, skipping.",
+    )
+    async def test_analysis(self):

```

Comment:
same here I think.
You'll see that this runs on images 307 and 308, where the one above runs on 307, 308 and 309,310 but only verifies the latter and the totals.